docs/astro/sql/operators/merge.rst

```diff
@@ -0,0 +1,143 @@
+.. _merge_operator:
+
+======================================
+merge operator
+======================================
+
+When to use the ``merge`` operator
+----------------------------------
+
+Merge operator is used when you expect some conflicts while merging two tables due to underline constraints. If you don't expect any conflicts you can use :ref:`append_operator`. There are two common cases when merging tables.
+
+Prerequisite
+------------
+
+Merge operator internally run different SQL queries based on the databases and some databases only allow you to run these queries if there are constraints on columns specified in parameter ``target_conflict_columns``. Below is the list of databases and their constraint requirement.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Database
```

Comment:
It is a bit confusing to have Database as one of the items of this list at the same level of the others.

Reply:
It's header row actually. Will be rendered like -
![Screenshot 2022-08-11 at 6 23 09 PM](https://user-images.githubusercontent.com/13021213/184137769-1e58b64c-9691-44ae-afc5-35034fe4bef7.png)

Reply:
🙈 that's great, @utkarsharma2, thanks!